### DISCUSSION TITLE
Raspberry Pi Alternative

### DISCUSSION THREAD

**Discussion by nixus619** [riffnshred/nhl-led-scoreboard#354] [Hardware]
Hey all,

Has anyone has success using something other than a raspberry pi?vim trying to make a few builds for some friends and I haven't been able to find a A+, B+, or even the zero 2 anywhere so I was wondering if there are any other options.

Thanks!

**Comment by riffnshred** [comment]
Not that I know of. There is a way to run it on your PC using this library https://github.com/ty-porter/RGBMatrixEmulator. But if you mean using raspberry pi like computers that has GPIO pins and all, I don't think so sadly.
